### DISCUSSION TITLE
Dormant user automated report

### DISCUSSION THREAD

**Discussion by satya2p** [community/community#141585] [Enterprise]
### Select Topic Area

General

### Body

is there an API to get this dormant user report scheduled and automated https://github.com/enterprises/footlocker/settings/compliance#reports

**Comment by BDadmehr0** [comment]
Hi!

From what I understand, you're looking to automate the dormant user report in your enterprise settings. Currently, GitHub doesn’t provide a direct API for scheduling or automating compliance reports like the dormant user report.

However, you can explore the following options:

1. GitHub API: While there isn't a specific API for compliance reports, you can use the GitHub REST API to fetch information about users in your organization (like activity levels) and create your own custom report. You could then automate this process with a cron job or scheduled script.
2. Third-Party Tools: Some external tools and integrations allow for more automated report generation. You might want to explore if any of these tools can handle compliance reporting for dormant users.
3. Manual Scheduling: If you need to stick with the built-in GitHub functionality, you could schedule reminders to manually generate and download the dormant user report from the compliance settings.

If none of these options meet your needs, you could also reach out to GitHub support or check the GitHub roadmap to see if API functionality for compliance reporting is planned in the future.

**Reply by satya2p** [reply]
I looked at org user activity API but adds more complexity when user are part of multi org. Do you have any recommendation for Third-Party Tools ?

**Reply by BDadmehr0** [reply]
You can use **Jira Service Management** to create reports and track user activity, pairing this with Atlassian Access for identity and access management.
